README.md

```diff
@@ -27,11 +27,49 @@ TODO: Write documentation for how to build initrd locally
 
 See [Hacking](HACKING.md)
 
-# Integrating with Kernel Snap
+# Releasing
 
-When your initrd DEB package is built, you need to perform following steps to make it a part of kernel snap:
+The UC initrd is included in kernel snaps. However, the process to get
+it inside the kernel snap is not immediate and consists of a few
+steps. First, we need to build the `ubuntu-core-initramfs`
+debian package in the [snappy-dev/image
+PPA](https://launchpad.net/~snappy-dev/+archive/ubuntu/image) by
+following these steps:
 
-TODO: Write documentation for how to integrate initrd.img to your (custom or canonical) kernel 
+1. Update the changelog with latest changes since last release (use `dch -i` for this)
+1. Commit using `debcommit --release -a` which will commit the changelog update & tag the release
+1. Propose a PR to the repo with the new changelog, get it reviewed and merged
+1. Push the tag to the repository with the new version (GitHub pull requests do not update tags)
+1. Build the source package by running
+
+        git clean -ffdx
+        gbp buildpackage -S -sa -d --git-ignore-branch
```

Comment:
can't you just use `--git-ignore-new` ? also `--ffdx` is destructive if you have unrelated files in the tree, might be nice to call out that this command will delete those files

Reply:
I've added a comment so it is more clear that git clean is destructive. But I think we don't want `--git-ignore-new` here: these are the instruction for releasing the package, not for just building it, so it will be good if we get an error if there are uncommited changes.

Reply:
@anonymouse64 one must really have a clean tree, if one builds a source package with unrelated files in place, one can trivially end up with them in the source package and like accidentatly have a 10GB large orig.tar.gz which contains some UC20 VM image.

Reply:
hmm that's not been my experience if you use `--git-ignore-new`, I have unrelated files in my snapd git tree but when I build the snapd deb and upload it we don't get all the random stuff in my git tree locally in the package that is uploaded

but sure in any case as long as it's documented that whoever is building this package needs to be sure that they have unrelated files removed, then this is fine with me